.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,95 @@
+name: Bug Report
+description: Create a bug report to help us improve
+title: "[Bug]: "
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to fill out this bug report!
```

According to https://www.collinsdictionary.com/dictionary/english/fill-out 'fill out' is mainly US. 
```suggestion
      value: Thank you for taking the time to complete this bug report!
```